src/Magnum/Math/Intersection.h

```diff
@@ -428,6 +444,15 @@ template<class T> bool rangeFrustum(const Range3D<T>& range, const Frustum<T>& f
     return true;
 }
 
+template<class T> bool rayRange(const Vector3<T>& rayOrigin, const Vector3<T>& invRayDir, const Range3D<T>& range) {
+    const Vector3<T> t0 = (range.min() - rayOrigin)*invRayDir;
+    const Vector3<T> t1 = (range.max() - rayOrigin)*invRayDir;
+    const Vector3<T> tmin = min(t0,t1);
+    const Vector3<T> tmax = max(t0,t1);
```

Comment:
[minmax()](https://doc.magnum.graphics/magnum/namespaceMagnum_1_1Math.html#a1be28fa49728193f3891a80e2e532f3a) is one operation less ;)